zh/pd-recover.md

```diff
@@ -0,0 +1,210 @@
+---
+title: 使用 PD Recover 恢复 PD 集群
+summary: 使用 PD Recover 恢复 PD 集群
+category: reference
+---
+
+# 使用 PD Recover 恢复 PD 集群
+
+[PD Recover](https://pingcap.com/docs-cn/stable/reference/tools/pd-recover) 是对 PD 进行灾难性恢复的工具，用于恢复无法正常启动或服务的 PD 集群。
+
+## 下载 pd-recover
+
+下载 TiDB 官方安装包：
+
+{{< copyable "shell-regular" >}}
+
+```
+wget https://download.pingcap.org/tidb-${version}-linux-amd64.tar.gz
+```
+
+`${version}` 是 TiDB 集群版本，例如，`v4.0.0-rc`。
+
+解压安装包：
+
+{{< copyable "shell-regular" >}}
+
+```
+tar -xzf tidb-${version}-linux-amd64.tar.gz
+```
+
+`pd-recover` 在 `tidb-${version}-linux-amd64/bin` 目录下。
+
+## 使用 pd-recover 恢复 PD 集群
+
+### 获取 Cluster ID
+
+{{< copyable "shell-regular" >}}
+
+```
+kubectl get tc ${cluster_name} -n ${namespace} -o='go-template={{.status.clusterID}}{{"\n"}}'
+```
+
+示例：
+
+```
+kubectl get tc test -n test -o='go-template={{.status.clusterID}}{{"\n"}}'
+6821434242797747735
+```
+
+### 获取 Alloc ID
+
+Alloc ID 是 TiKV 或者 TiFlash 分配的 StoreID。
```

Comment:
alloc-id is generated by a global id allocator, it's used in store id, peer id, region id etc. So we cannot use the store id to decide the current alloc id.

Reply:
updated